cocotb/share/makefiles/simulators/Makefile.questa

```diff
@@ -131,7 +140,9 @@ EXTRA_LIBDIRS := -L$(MODELSIM_BIN_DIR)
 OLD_PATH := $(shell echo "$(PATH)" | sed 's/(/\\(/g' | sed 's/)/\\)/g' | sed 's/ /\\ /g')
 
 LIB_LOAD := PATH=$(MINGW_BIN_DIR):$(OLD_PATH):$(LIB_DIR)
-NEW_PYTHONPATH := $(shell echo "$(PYTHONPATH)" | sed -e 's/\\/\//g' -e 's/\([a-zA-Z]\):\//\/\1\//g' -e 's/;/:/g')
+NEW_PYTHONPATH:=$(shell python -c "import sys, os; print(':'.join(['/'+dir.replace(os.sep,'/').replace(':','') for dir in sys.path]))")
```

Comment:
This could really do with a comment. I assume it's the same as all the other makefiles?

Reply:
Would `print('os.pathsep.join(subprocess.check_output(['cygpath', '-m', d]) for d in sys,path))` work here?

Reply:
Can do but in practice it will be something liek this:

`python -c "import sys, os, subprocess; print(os.pathsep.join(str(subprocess.check_output(['cygpath', '-m', d]))[2:-3] for d in sys.path if d != '' ))"`

Is that better?